srcpkgs/discord/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'discord'
 pkgname=discord
 version=0.0.25
-revision=1
+revision=2
 archs="x86_64"
 depends="alsa-lib dbus-glib gtk+3 libnotify nss libXtst libcxx libatomic
- xdg-utils webrtc-audio-processing"
+ xdg-utils webrtc-audio-processing gst-plugins-bad1"
```

Comment:
```suggestion
 xdg-utils webrtc-audio-processing libXScrnSaver"
```
I went through all the dependencies of `gst-plugins-bad1`, and the missing one is `libXScrnSaver`.

further confirmed with `strace`:
```
$ strace -Z -ff -o discord.log Discord
$ grep libXss discord.log.* | wc -l
384
$ sudo xbps-install libXScrnSaver
$ strace -Z -ff -o discord-xss.log Discord
$ grep libXss discord-xss.log.* | wc -l
1
```